WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -1010,14 +1015,18 @@ decisions = {
 			independent = yes
 			OR = {
 				has_landed_title = k_persia
+				any_realm_lord = { has_landed_title = k_persia }
 				has_landed_title = k_khiva
+				any_realm_lord = { has_landed_title = k_khiva }
 				has_landed_title = k_afghanistan
+				any_realm_lord = { has_landed_title = k_afghanistan }
 				has_landed_title = k_baluchistan
+				any_realm_lord = { has_landed_title k_baluchistan }
 				has_landed_title = e_persia
 				culture = sogdian
 			} #change for SWMH
 			prestige = 250
-			realm_size = 60
+			realm_size = 30
```

Comment:
There shouldn't be size requirements, only wealth ones and actual provinces on the Silk Road, but thirty is plenty.